### DISCUSSION TITLE
What should I do to highlight the selected mesh?

### DISCUSSION THREAD

**Discussion by devweilog** [visgl/deck.gl#5663] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17996469/114696051-d6b30b00-9d4e-11eb-8c42-6437ed8b20ae.png)

As you can see, this is what I want to achieve in the end. 

To do this, I had to complete two steps.
The first step is to use some software to mark a building's mesh with the same tag. 
The second step is when I click on a mesh, I query all the relevant mesh through the tag and then highlight them.

Now I can  get the selectedTiles by layer's click event,but the question is, how can I highlight the mesh contained in the selectedTiles?

**Comment by ibgreen** [comment, marked as answer]
I assume you are using I3S or 3D tiles? Selection and highlighting for I3S objects and tiles is currently being developed in loaders.gl. The `i3s-debug` example currently contains forked deck.gl layers to implement this (you could certainly copy these to your code, however we will soon start porting the changes back to deck.gl hoping to get things in place over the coming month).

Example of tile picking (object picking is implemented but not in the UI.

https://loaders.gl/examples/i3s-debug

<img width="552" alt="Screen Shot 2021-04-14 at 11 14 21 AM" src="https://user-images.githubusercontent.com/7025232/114759093-96ce3280-9d12-11eb-9df5-c518e3ef56e8.png">

@belom88 @dryabinin-actionengine

**Reply by devweilog** [reply]
Oh, that helps me a lot!😺😺
Thank you very much, I'm also looking forward to your new features.

**Reply by belom88** [reply]
Object/feature picking is in general i3s application example:
![image](https://user-images.githubusercontent.com/18549629/114865649-3c5cd100-9dfb-11eb-9d2f-0733d1077b28.png)

**Reply by belom88** [reply]
https://loaders.gl/examples/i3s
